lesson_10/solid/solid_app/src/main/java/com/codedifferently/lesson10/library/MediaItem.java

```diff
@@ -0,0 +1,31 @@
+package com.codedifferently.lesson10.library;
+
+import com.codedifferently.lesson10.library.exceptions.MediaItemCheckedOutException;
+
+public abstract class MediaItem {
```

Comment:
I expect to see this integrated with the `Library` class so that instead of working directly with `Book` instances, it works with the `MediaItem` interface instead. Please implement.